addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -444,39 +485,83 @@ a.no-decoration {
         max-height: 90vh;
     }
 
+    .carousel-control-prev .fa {
+        padding-right: 2px;
+    }
+    .carousel-control-next .fa {
+        padding-left: 2px;
+    }
+
     .carousel-control-prev, .carousel-control-next {
         height: 70%;
         top: 15%;
         opacity: 0.5;
         cursor: pointer;
+        transition: opacity 0.8s;
+
         &:focus {
             opacity: 0.65;
         }
         &:hover {
             opacity: 0.8;
         }
         > span {
-            background: rgba(0, 0, 0, 0.8);
+            width: 2.5rem;
+            height: 2.5rem;
+            line-height: 2.5rem;
+            color: black;
+            background: white;
+            font-size: 1.15rem;
+            border: 1px solid $gray-400;
```

Comment:
Use `gray('400')` instead of `$gray-400`. This is not important because I made it work but in a standard BS project this could be buggy depending on how it is configured, so I prefer not to encourage it.